hapi-fees/src/main/java/com/hedera/services/usage/schedule/ScheduleOpsUsage.java

```diff
@@ -61,7 +61,7 @@ public FeeData scheduleCreateUsage(TransactionBody scheduleCreate, SigUsage sigU
 		var op = scheduleCreate.getScheduleCreate();
 
 		var scheduledTxn = op.getScheduledTransactionBody();
-		long msgBytesUsed = scheduledTxn.getSerializedSize() + op.getMemoBytes().size();
+		int msgBytesUsed = scheduledTxn.getSerializedSize() + op.getMemoBytes().size();
```

Comment:
`msgBytesUsed` must be `long` because on line 83, we will do `estimate.addBpt(msgBytesUsed);` which requires a `long`. Here a `cast` is unavoidable!

Reply:
Fixed.